project/test/resources/OPConsultantRecord-Output.json

```diff
@@ -1,2688 +1,2143 @@
+          "code": "8302-2",
+          "display": "Body height"
+        } ],
+        "text": "Height"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "effectiveDateTime": "2024-07-01T16:29:32+05:30",
+      "valueQuantity": {
+        "value": 160.0,
+        "unit": "cm",
+        "system": "http://unitsofmeasure.org",
+        "code": "cm"
+      }
+    }
+  }, {
+    "fullUrl": "urn:uuid:709e85da-9ded-4755-912d-6b03286a569c",
+    "resource": {
+      "resourceType": "Observation",
+      "id": "709e85da-9ded-4755-912d-6b03286a569c",
+      "meta": {
+        "versionId": "0",
+        "lastUpdated": "2024-07-01T16:29:32.572+05:30",
+        "profile": [ "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Observation" ]
+      },
+      "status": "final",
+      "category": [ {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "vital-signs",
+          "display": "Vital Signs"
+        } ],
+        "text": "Vital Signs"
+      } ],
+      "code": {
+        "coding": [ {
+          "system": "http://loinc.org",
+          "code": "29463-7",
+          "display": "weight"
+        } ],
+        "text": "Weight"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "effectiveDateTime": "2024-07-01T16:29:32+05:30",
+      "valueQuantity": {
+        "value": 62.0,
+        "unit": "kg",
+        "system": "http://unitsofmeasure.org",
+        "code": "kg"
+      }
+    }
+  }, {
+    "fullUrl": "urn:uuid:64482fc0-c5c0-4f49-8dc7-8632ea71c6db",
+    "resource": {
+      "resourceType": "Observation",
+      "id": "64482fc0-c5c0-4f49-8dc7-8632ea71c6db",
+      "meta": {
+        "versionId": "0",
+        "lastUpdated": "2024-07-01T16:29:32.573+05:30",
+        "profile": [ "https://nrces.in/ndhm/fhir/r4/StructureDefinition/Observation" ]
+      },
+      "status": "final",
+      "category": [ {
+        "coding": [ {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "vital-signs",
+          "display": "Vital Signs"
+        } ],
+        "text": "Vital Signs"
+      } ],
+      "code": {
+        "coding": [ {
+          "system": "http://loinc.org",
+          "code": "39156-5",
+          "display": "Body mass index (BMI) [Ratio]"
+        } ],
+        "text": "Body Mass Index"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "effectiveDateTime": "2024-07-01T16:29:32+05:30",
+      "valueQuantity": {
+        "value": 62.0,
+        "unit": "kg/m2",
+        "system": "http://unitsofmeasure.org",
+        "code": "kg/m2"
+      }
+    }
+  }, {
+    "fullUrl": "urn:uuid:9d7e0de1-e040-42ea-a246-76aa6d85e34b",
```

Comment:
This observation is for Blood Group and is not a observation for Vital signs. You don't need to specific the category.